packages/morphic-io-ts-interpreters/src/hkt.ts

```diff
@@ -1,4 +1,6 @@
-import type { Type } from 'io-ts'
+import type { Errors, Type } from 'io-ts'
+import { Either, map } from 'fp-ts/lib/Either'
```

Comment:
This brings in a lot from either it would be better to locally implement .map & only bring in the type

Reply:
removing reliance on third party might come later @mikearnaldi